Makefile

```
@@ -10,7 +10,14 @@ apfs-y := btree.o compress.o dir.o extents.o file.o inode.o key.o message.o \
 	  namei.o node.o object.o spaceman.o super.o symlink.o transaction.o \
 	  unicode.o xattr.o xfield.o
 
-default:
-	make -C /lib/modules/$(KERNELRELEASE)/build M=$(shell pwd)
+KDIR ?= /lib/modules/`uname -r`/build
```

```suggestion
KDIR ?= /lib/modules/$(KERNELRELEASE)/build
```
I would also suggest defining this variable directly below the definition of `KERNELRELEASE`.